module/VuFind/src/VuFind/Db/Row/UserList.php

```diff
@@ -305,4 +308,118 @@ public function delete($user = false, $force = false)
         // Remove the list itself:
         return parent::delete();
     }
+
+    /**
+     * Get identifier (returns null for an uninitialized or non-persisted object).
+     *
+     * @return ?int
+     */
+    public function getId(): ?int
+    {
+        return $this->id ?? null;
+    }
+
+    /**
+     * Title setter
+     *
+     * @param string $title Title
+     *
+     * @return UserListEntityInterface
+     */
+    public function setTitle(string $title): UserListEntityInterface
+    {
+        $this->title = $title;
+        return $this;
+    }
+
+    /**
+     * Get Title.
+     *
+     * @return string
+     */
+    public function getTitle(): string
+    {
+        return $this->title;
+    }
+
+    /**
+     * Description setter
+     *
+     * @param string $description Description
+     *
+     * @return UserListEntityInterface
+     */
+    public function setDescription(string $description): UserListEntityInterface
+    {
+        $this->description = $description;
+        return $this;
+    }
+
+    /**
+     * Get Description.
+     *
+     * @return string
+     */
+    public function getDescription(): string
+    {
+        return $this->description;
+    }
+
+    /**
+     * Created date setter.
+     *
+     * @param DateTime $dateTime Created date
+     *
+     * @return UserListEntityInterface
+     */
+    public function setCreated(DateTime $dateTime): UserListEntityInterface
+    {
+        return $this->created = $dateTime;
+    }
+
+    /**
+     * Get created date.
+     *
+     * @return DateTime
+     */
+    public function getCreated(): DateTime
+    {
+        return $this->created;
+    }
+
+    /**
+     * Set the list public.
+     *
+     * @param bool $public Created date
+     *
+     * @return UserListEntityInterface
+     */
+    public function setIsPublic(bool $public): UserListEntityInterface
+    {
+        $this->public = $public ? 1 : 0;
+        return $this;
+    }
+
+    /**
+     * Check if the list is public
+     *
+     * @return bool
+     */
+    public function getPublic(): bool
```

Comment:
I think it probably makes sense to just delete this and keep the existing isPublic() method, and then change getPublic() to isPublic() in the interface. It looks like the method is already named isPublic both here and in #2233, and both implementations are compatible with each other... so I don't think there's a reason to introduce a different name here.